ros/src/tl_detector/light_classification/ssd_tl_classifier.py

```diff
@@ -34,37 +47,112 @@ def load_graph(graph_file):
                 serialized_graph = fid.read()
                 od_graph_def.ParseFromString(serialized_graph)
                 tf.import_graph_def(od_graph_def, name='')
-
         return graph
 
+    def filter_boxes(self, boxes, scores, classes):
+        """
+        Filters boxes with scores less than
+        confidence threshold
+        """
+        n = len(classes)
+        idxs = []
+        for i in range(n):
+            if scores[i] >= self.confidence:
+                idxs.append(i)
+
+        boxes = boxes[idxs, ...]
+        scores = scores[idxs, ...]
+        classes = classes[idxs, ...]
+        return boxes, scores, classes
+
+    def _get_debug_image(self, image, boxes, scores, classes):
+        """Draws detected bounding boxes"""
+        if len(classes) == 0:
```

Comment:
Automatic code reviewer CodeFactor said that writing `if not classes: ...` is more pythonic in this case.

Reply:
`if not classes:` doesn't work with numpy, it raises "ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()"......used `if classes.size == 0` instead.